src/main/java/mod/crend/movesubtitles/mixin/SubtitlesHudMixin.java

```diff
@@ -24,20 +24,27 @@ public abstract class SubtitlesHudMixin {
 	@WrapOperation(method = "render", at=@At(value = "INVOKE", target = "Lnet/minecraft/client/util/math/MatrixStack;translate(FFF)V"))
 	private void moveSubtitles(MatrixStack instance, float x, float y, float z, Operation<Void> original, DrawContext context, @Local(ordinal = 0) int i, @Local(ordinal = 1) int j) {
 		int centerX = j / 2;
-		int heightPerColumn = 9;
+		int heightPerRow = 10;
```

Comment:
Hm, that was a silly mapping mistake on my part. Thanks!
As for why it was set to 9 and always incremented by one, I did that because it matches the decompiled code in the mixin'd method.

Reply:
Oh I see that now. Would you prefer `heightPerRow` changed to `9 + 1` to reflect that then?

Reply:
No, it does make sense like this as well.